### DISCUSSION TITLE
cursor: not-allowed;

### DISCUSSION THREAD

**Discussion by ukqfhf** [VKCOM/VKUI#6180] [Ideas] [answered]
Почему бы не выставлять `cursor: not-allowed` для disabled компонентов?

Стоит отметить что далеко не все киты так делают. 

Поддерживают: Ant Design, Baseweb;
Не поддерживат: MUI;

Референсы:
- [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/cursor)
- [csswg](https://drafts.csswg.org/css-ui/#cursor)

_PS: тему можно расширить и на другие значения свойства cursor_

**Comment by ukqfhf** [comment]
Не нашел информации рекомендуется ли выставлять cursor с точки зрения a11y. Если есть информация, то поделитесь пожалуйста 🙏

**Comment by SevereCloud** [comment, marked as answer]
Добавил в [6132](https://github.com/VKCOM/VKUI/pull/6132/files#diff-c0338892316785a00fde5b26ea27c16100054b94334e7d3720d2dd1c9225d9b9R16-R19)
